.changeset/wicked-kangaroos-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sveltejs/kit": patch
+---
+
+fix: correctly handle aliases to files in the `./svelte-kit` directory
```

Comment:
`./svelte-kit` doesn't exist. it would be `./.svelte-kit`. but easier to just write `.svelte-kit` here
```suggestion
fix: correctly handle aliases to files in the `.svelte-kit` directory
```